nise/__init__.py

```diff
@@ -1,3 +1,3 @@
-__version__ = "4.0.2"
+__version__ = "4.0.3"
```

Comment:
Because these changes only affect docs, we do not _need_ to bump the version here. These docs don't go to pypi or anywhere outside of github.